include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid.
+   *
+   * Following alignment conditions have to be fulfilled: $MB_A$ = $NB_B$ and $NB_A$ = $MB_B$
```

Comment:
Say "The following ...". I had to think about whether "following" is a verb or adjective. Same in a few places below.

Also end the sentence with a period.

Reply:
You beat me to it! I was also going to make the suggestion that the style of all of the documentation be made more consistent with that of the rest of the class. The conditions can also be put in a `@note`.